dockerfiles/init/manifests/codenvy.pp

```diff
@@ -303,6 +303,12 @@
   $db_schema_flyway_scripts_locations=getValue("DB_SCHEMA_FLYWAY_SCRIPTS_LOCATIONS","classpath:che-schema,classpath:codenvy-schema")
 
 ###############################
+# Rsync backup/restore configuration
+  $rsync_restore_bwlimit=getValue("RSYNC_RESTORE_BWLIMIT","7000")
```

Comment:
I don't say that we definitely will. And don't know when it is possible. It is possible that we will use completely other system of configuring that. So just renaming could not be the option. 
The reason why I think it is possible is that we are going to allow user sync different folders of workspace, not only projects. And if user chooses to sync 20GB .m2 folder it may take a lot of time.